### ISSUE TITLE
Public gorae not displayed on public page

### ISSUE THREAD

**Issue by rivmud-fabwen** [dalten-collective/gora#4]
Having an issue with the public gora page.  When I load the page it says "Owned %gora Click the preview image to see more details." but there's nothing visible below that.  

In the dev tools I see that it does have the gora images and if I remove `display: none` from the modal elements, I can see them.  

I'm pretty sure it's just a display bug.

![image](https://user-images.githubusercontent.com/67772522/151642977-b401246c-e8fe-459d-b209-e4ba9bf58022.png)

**Comment by rabsef-bicrym** [comment]
I believe this is a resolved issue - could you please confirm @rivmud-fabwen

**Comment by rivmud-fabwen** [comment]
Still broken for me @rabsef-bicrym

**Comment by rabsef-bicrym** [comment]
can you confirm you have a gora that has the property `host-public` as `%.y` and that that gora (by id) also appears in the `state` element `my-public`?

You should be able to get
```
:gora +dbug [%state %my-public]
```

or, I think you could even just 
```
.^((list [@t @t]) %gy /=gora=/pub)
```
and post that?

**Comment by rivmud-fabwen** [comment]
```
>   {0v1.f61ef.1nicr.6kbem.ae0nf.md4d1 0v7.t96mu.fiu1h.4egjs.4e32f.gbp24}
> :gora +dbug [%state %my-public]
```

**Comment by rabsef-bicrym** [comment]
could you do that scry plz?

**Comment by rabsef-bicrym** [comment]
should be like 
```
> .^((list [@t @t]) %gy /=gora=/pub)
~[['Assembly 2021' 'https://freedom-club.sfo2.digitaloceanspaces.com/rabsef-bicrym/2021.10.17..21.05.39-image.png']]
```

**Comment by rivmud-fabwen** [comment]
It just prints a `~`
```
> .^((list [@t @t]) %gy /=gora=/pub)
~
~rivmud-fabwen:dojo>
```

**Comment by rabsef-bicrym** [comment]
Ok - that would lead me to believe that neither of the gorae you've marked as public are public according to the host - that's what that should be showing.

When you said "In the dev tools I see that it does have the gora images and if I remove display: none from the modal elements, I can see them." do you mean you can see the gorae that showed in that state debug call but which are absent in the /pub scry?

**Comment by rivmud-fabwen** [comment]
Correct, the dev tools show the source images for 2 gorae that I made public.  Also if I make one of them private, it no longer shows up in the dev tools.

**Comment by rabsef-bicrym** [comment]
Thank you - I understand your description of the issue now. Definitely a bug.
